kotlin/workflow-core/src/main/java/com/squareup/workflow/WorkflowContext.kt

```diff
@@ -175,6 +175,20 @@ fun <StateT : Any, OutputT : Any, ChildStateT : Any, ChildOutputT : Any, ChildRe
     ): ChildRenderingT = compose(child, Unit, key, handler)
 // @formatter:on
 
+/**
+ * Convenience alias of [WorkflowContext.compose] for workflows that don't take input or emit output.
```

Comment:
Didn't you already put this change up in another PR?

Reply:
There are a bunch, I added one more.